demos/device_defender_for_aws/core_mqtt_config.h

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
This file is not needed, it already exists in the board's config files folder

Reply:
Correct, will remove it.